release_notes/370-release-notes.adoc

```diff
@@ -6,7 +6,8 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-*Release date:* 2 June 2022
```

Comment:
I notice the release date format is different in https://github.com/openshift/openshift-docs/pull/46670/files#diff-4a473591150b2cdd6f72b2aca9827461644a367c506e9ee6a979fed45c38837c. Is this intentional? We might want to standardize across release notes.

Reply:
I had checked the Style Guide and noticed that the format is DD - Month - Year, so I changed it on the latest ones, but didn't go back and change the older ones. Do you think it's ok to just change going forward?